targets/challenge/challenge-textures.qrc

```diff
@@ -1,17 +1,17 @@
-<!DOCTYPE RCC><RCC version="1.0">
-<qresource>
-	<file>textures/feeder.png</file>
-	<file>textures/feederr.png</file>
-	<file>asebachallenge_fr.qm</file>
-	<file>asebachallenge_de.qm</file>
-	<file>asebachallenge_es.qm</file>
-	<file>asebachallenge_it.qm</file>
-	<file>asebachallenge_ja.qm</file>
-	<!-- insert translation here (DO NOT REMOVE) -->
```

Comment:
I didn't want to do all this..  
I just use QTcreator to add files in the .qrc... why original tool is not working? 
I is painful to correct thing done behind...